test/core/tokenHandler/test.js

```diff
@@ -55,32 +53,124 @@ define(['core/promise', 'core/tokenHandler', 'jquery.mockjax'], function(Promise
         );
     });
 
-    QUnit.module('behaviour');
+    QUnit.module('behaviour', {
+        beforeEach: function() {
+            this.cachedModuleConfig = requirejs.s.contexts._.config.config['core/tokenHandler'];
+        },
+        afterEach: function() {
+            requirejs.s.contexts._.config.config['core/tokenHandler'] = this.cachedModuleConfig;
```

Comment:
I'm a little surprised it works to mess about with the internal data of requirejs, but it seems ok!

Reply:
Yes :( unfortunately, this a necessary mess
BTW, you can find another place where the same "mess" is applied: https://github.com/oat-sa/tao-core-sdk-fe/blob/8cf13ba0c121d845d44c6e675084920cabacf6ef/src/core/router.js#L216